src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.TestLibrary/System.Text.Json.TestLibrary.targets

```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <DefaultReferenceExclusion Include="System.Text.Json.SourceGeneration" />
-    <ProjectReference Include="..\..\src\System.Text.Json.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Text.Json\ref\System.Text.Json.csproj" />
```

Comment:
That's incorrect. A test project shouldn't reference a reference source project.

Reply:
```suggestion
    <ProjectReference Include="..\..\src\System.Text.Json.csproj" />
```